lib/Staticman.js

```diff
@@ -573,18 +573,18 @@ class Staticman {
   }
 
   processMerge (fields, options) {
-    this.fields = Object.assign({}, fields)
-    this.options = Object.assign({}, options)
-
+    /*
+     * Requests to process a merge will only come via a webhook, not a user. As such, no need to
+     * use the ErrorHandler when returning errors.
+     */
     return this.getSiteConfig().then(config => {
       const subscriptions = this._initialiseSubscriptions()
 
-      return subscriptions.send(options.parent, fields, options, this.siteConfig)
+      return subscriptions.send(options.parent, fields, options, this.siteConfig).catch(err => {
+        return Promise.reject(err)
+      })
```

Comment:
```suggestion
      return subscriptions.send(options.parent, fields, options, this.siteConfig)
```

I don't think this catch block is necessary since any errors should thrown here will immediately get caught and handled by lines 586-587 below right?

Reply:
Yup. I added a new unit test to verify. Thanks.